uniffi_bindgen/src/bindings/python/gen_python/mod.rs

```diff
@@ -351,14 +351,14 @@ impl PythonCodeOracle {
             FfiType::Float64 => "ctypes.c_double".to_string(),
             FfiType::RustArcPtr(_) => "ctypes.c_void_p".to_string(),
             FfiType::RustBuffer(maybe_suffix) => match maybe_suffix {
-                Some(suffix) => format!("_UniffiRustBuffer{suffix}"),
-                None => "_UniffiRustBuffer".to_string(),
+                Some(suffix) => format!("UniffiRustBuffer{suffix}"),
+                None => "UniffiRustBuffer".to_string(),
```

Comment:
The underscores were actually _added_ in #1599

Reply:
Doh!

I just updated the PR to always use the underscores.  PEP 8 seems to say we don't need them if we use `__all__`, but the autocomplete example from 1599 is pretty convincing to me.

I also updated the names for a few more functions.  `_rust_call` is now `_uniffi_rust_call`